packages/next/taskfile-ncc.js

```diff
@@ -40,7 +40,9 @@ function writePackageManifest(packageName) {
 
   let typesFile = types || typings
   if (typesFile) {
-    typesFile = require.resolve(join(packageName, typesFile))
+    try {
+      typesFile = require.resolve(join(packageName, typesFile))
+    } catch (_) {}
```

Comment:
It's odd that this needs a try/catch, is a package we're consuming published incorrectly?
I'd rather see this one package whitelisted than swallowing all errors.

Reply:
Why would we error if the types file is missing, don't we want to check multiple places?

Reply:
The types file should never be missing.

Reply:
I see, it looks like it was providing a types directory and we weren't resolving the directory correctly